### ISSUE TITLE
Export-ArXivArchive should allow rewriting TeX commands

### ISSUE THREAD

**Issue by cgranade** [cgranade/posh-tex#8]
For some LaTeX build configurations, it may be convenient to redefine TeX commands for an arXiv submission from their values in a working repository. For instance, a ``\figurefolder`` command may point to ``tex/fig/`` in a working repository, but ``fig/`` in an arXiv archive. PoShTeX should support this usecase by allowing users to specify a set of commands whose values should change in the arXiv export. For instance, this enhancement is needed even for the example manifest in ``ArXiv.ps1``:

```powershell
$ExampleManifest = @{
    ProjectName = "foo";
    TeXMain = "tex/foo.tex";
    AdditionalFiles = @{
        "fig/*.pdf" = $null;
        "fig/*.png" = $null;
        "tex/*.sty" = "/";
        "fig/*.mp4" = "anc/";
    };
}
```

**Comment by ihincks** [comment]
Solved by #13 ?

**Comment by cgranade** [comment]
Ah, I forgot to close this, thanks for pointing it out!
